src/Http/WebUtilities/ref/Microsoft.AspNetCore.WebUtilities.netcoreapp.cs

```diff
@@ -161,10 +161,12 @@ public partial class HttpResponseStreamWriter : System.IO.TextWriter
         public override System.Threading.Tasks.Task FlushAsync() { throw null; }
         public override void Write(char value) { }
         public override void Write(char[] values, int index, int count) { }
+        public override void Write(System.ReadOnlySpan<char> values) { }
         public override void Write(string value) { }
         public override System.Threading.Tasks.Task WriteAsync(char value) { throw null; }
         public override System.Threading.Tasks.Task WriteAsync(char[] values, int index, int count) { throw null; }
```

Comment:
Should we add a parallel method for WriteAsync that accepts a ReadOnlyMemory<char>?

Reply:
I have #18451 for those, but it is based on this one, so I will mark it ready for review once this is merged. Unless you prefer to combine it.

Reply:
Ah, I see now!